protonvpn_cli/utils.py

```diff
@@ -180,6 +189,23 @@ def is_connected():
     return True if ovpn_processes != [] else False
 
 
+def wait_for_network():
+    """Check if internet access is working"""
+
+    print("Waiting for connection...")
+    while True:
+        try:
+            logger.debug("Waiting for connection...")
+            call_api("/test/ping", handle_errors=False)
+            time.sleep(2)
+            print("Connection working!")
+            logger.debug("Connection working!")
+            break
+        except (requests.exceptions.ConnectionError,
```

Comment:
Don't we want to exit after a while?

Reply:
It automatically stops after 20 minutes now